src/components/movie-view/movie-view.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { Button } from 'react-router-dom';
+import { Button } from 'react-bootstrap';
```

Comment:
This was causing an error because `react-router-dom` doesn't export a component called `Button`